lib/plugins/bed.js

```diff
@@ -2,32 +2,33 @@ const Vec3 = require('vec3').Vec3
 
 module.exports = inject
 
+const CARDINAL_DIRECTIONS = ['south', 'west', 'north', 'east']
+
 function inject (bot) {
   bot.isSleeping = false
 
   function parseBedMetadata (bedBlock) {
-    const metadata = {}
-    const bMetadata = bedBlock.metadata.toString(2).padStart(4, '0')
-
-    metadata.part = Number(bMetadata[0])
-    metadata.occupied = Number(bMetadata[1])
-    switch (bMetadata.slice(2, 4)) {
-      case '00':
-        metadata.facing = 'south'
-        metadata.headOffset = new Vec3(0, 0, 1)
-        break
+    const bitMetadata = bedBlock.metadata.toString(2).padStart(4, '0')
+
+    const metadata = {
+      part: parseInt(bitMetadata[0], 2),
+      occupied: parseInt(bitMetadata[1], 2),
```

Comment:
Why are you using binary there?

Reply:
Are you suggesting to use Booleans?

Reply:
no, just thinking why binary and not decimal?

Reply:
`bedBlock.metadata` returns a int and if I want to know what that number means I have to parse it to binary and interpret 
 each bit (https://minecraft.gamepedia.com/Bed). I don't know if there's any workaraund

Reply:
`bedBlock.metadata` returns a int and if I want to know what that number means I have to parse it to binary and interpret 
 each bit (https://minecraft.gamepedia.com/Bed). I don't know if there's any workaraund